src/avgScrabbleScoreFromFile.js

```diff
@@ -11,11 +11,16 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  * sample-words.txt.
  */
 function avgScrabbleScoreFromFile(fileName) {
+  let scrabbleArray=fs.readFileSync(`../src/${fileName}`,'utf-8').split('\n');
```

Comment:
You can just do `fs.readFileSync(fileName, 'utf-8')`.

There's a small bug here. Try testing this one a simple file that looks like

```
a
a
```

What would you expect the avg scrabble score to be? What do you get instead?